internal/cmd/skupper/site/kube/site_create.go

```diff
@@ -173,25 +183,37 @@ func (cmd *CmdSiteCreate) WaitUntil() error {
 	}
 
 	waitTime := int(cmd.timeout.Seconds())
+
 	err := utils.NewSpinnerWithTimeout("Waiting for status...", waitTime, func() error {
 
 		resource, err := cmd.Client.Sites(cmd.Namespace).Get(context.TODO(), cmd.siteName, metav1.GetOptions{})
 		if err != nil {
 			return err
 		}
 
-		if resource != nil && resource.IsConfigured() {
+		isStatusReached := false
+
+		switch cmd.status {
+		case "pending":
```

Comment:
I don't think 'pending' is a state you wait for. It is an indication that no state has been settled yet. One option is to wait until a 'configured' condition has been recorded, whether true or false (if false there is likely to be an error).

Reply:
That makes sense. The wait flag also allows to wait for `configured`, but following the Refdog documentation (https://skupperproject.github.io/refdog/commands/site/create.html), when a user indicates `--wait pending` it could mean that they don't expect to wait at all, right? so in that case I just need to skip the WaitUntil step. Do you agree with that?

Reply:
I'm disagreeing with the refdog documentation in that I don't think 'pending' should be one of the options to wait. @ssorj

Reply:
I have removed the `pending` value (d917edfd92b141691e186b64e06e94321787702f) from the possible values to include in the `--wait` flag, in order to unblock this pull request until until we find a consensus.